docs/60_initialize_vendors.md

```diff
@@ -0,0 +1,473 @@
+
+Populate the following variables in the `Variables` section of the `Workspace`:
+
+1. `AWS_ACCESS_KEY_ID`
+2. `AWS_SECRET_ACCESS_KEY`
+3. `HEROKU_API_KEY`
+4. `HEROKU_EMAIL`
+
+<br/><br/>
+<img
+src="../img/terraform_vars.png"
+alt="retrieve"
+style="width: 60%; height: auto; display: block; margin-left: auto;  margin-right: auto;"/>
+<br/><br/>
+
+##### Link GitHub Repository
+
+Next, link your appropriate GitHub repository, in this case, most likely your fork of [dandi-infrastructure](https://github.com/dandi/dandi-infrastructure).
+
+<br/><br/>
+<img
+src="../img/tf_vcs.png"
+alt="retrieve"
+style="width: 60%; height: auto; display: block; margin-left: auto;  margin-right: auto;"/>
+<br/><br/>
+
+Link your repository -- you may need to declare the appropriate subdirectory 
+
+<br/><br/>
+<img
+src="../img/tf_repo.png"
+alt="retrieve"
+style="width: 60%; height: auto; display: block; margin-left: auto;  margin-right: auto;"/>
+<br/><br/>
+
+You can now invoke Terraform `plan` and `apply` from the Terraform Cloud UI or via your GitHub repository Actions.
+
+## Netlify
+
+The frontend for `dandi-archive` is served via [Netlify](https://www.netlify.com/)
+
+##### Create Account and Initialize Project
+
+First, create an account. After creating an account, you'll want to navigate to `Sites`, where you can `Add a new site`, and then `Import an existing project`
+
+<br/><br/>
+<img
+src="../img/start_netlify.png"
+alt="retrieve"
+style="width: 60%; height: auto; display: block; margin-left: auto;  margin-right: auto;"/>
+<br/><br/>
+
+You'll want to next connect `dandi-archive` via `Deploy with GitHub`. Enable Netlify to be authorized as a GitHub app. Once you have
+enabled authorization, you'll need to specifically link your appropriate repositories:
+
+<br/><br/>
+<img
+src="../img/netlify_auth.png"
+alt="retrieve"
+style="width: 60%; height: auto; display: block; margin-left: auto;  margin-right: auto;"/>
+<br/><br/>
+
+Now once you see the appropriate repository, you'll want to navigate to configure where Netlify can find and build your site
+
+<br/><br/>
+<img
+src="../img/netlify_repo_search.png"
+alt="retrieve"
+style="width: 60%; height: auto; display: block; margin-left: auto;  margin-right: auto;"/>
+<br/><br/>
+
+For usage of Netlify, one could refer to declaring a `netlify.toml` configuration file [like the one referenced in DANDI Archive](https://github.com/dandi/dandi-archive/blob/master/web/netlify.toml)
+
+These values can also be replicated in the settings. 
+
+<br/><br/>
+<img
+src="../img/netlify_config.png"
+alt="retrieve"
+style="width: 60%; height: auto; display: block; margin-left: auto;  margin-right: auto;"/>
+<br/><br/>
+
+Your frontend should be able to deploy to an auto-generated URL via Netlify now! Steps for domain management and configuration are described further in the [Frontend Deployment](../64_dandi_archive/#frontend-deployment) section of these within the DANDI Archive setup.
```

Comment:
Question came up on knowing how many minutes is needed by netlify for DANDI instance.

Reply:
When you say `minutes` in this context for Netlify, do you mean "build minutes"? (e.g. how long Netlify runners are required to run to deploy?) Or something else?